### ISSUE TITLE
Add global nightlights data

### ISSUE THREAD

**Issue by Schpidi** [eurodatacube/eodash#789]
https://blogs.worldbank.org/opendata/light-every-night-new-nighttime-light-data-set-and-tools-development

**Comment by santilland** [comment]
From what i can read in the documentation (https://worldbank.github.io/OpenNightLights/wb-light-every-night-readme.html) i only see direct access to the data hosted on aws and there are multiple variants that can be fetched (see list at the end).
I think the first step would be to fetch the wanted datasets and load them to the sentinelhub mundi instance as we have been doing with other datasets.
Afterwards we could include the data as we have done for other layers.

```
.vis.co.tif: a COG with a single band containing the visible nighttime imagery (this is the key data file)

Data range: [0, 63]

“No-data” value: 255

Data type: byte

Units: Digital Number (DN) or unitless

.vis.co.json: a JSON file following the STAC schema with metadata for orbital segment, including a timestamp and a geospatial bounding box

.flag.co.tif: a COG with a single band containing bitflags that can be queried for data quality

Data range: [0,215-1]

“No-data” value: 215

Bit-fields: see Table 2

Data type: 16-bit integer

Units: Unitless

.tir.co.tif: a COG with a single band containing the OLS Thermal Infrared (TIR) data

Data range: [0, 255]

“No-data” value: 255 (note this is not unique, use vis.co.tif or another companion file to confirm areas of no data)

Data type: byte

Units: Scaled byte data. These values can be converted to temperature in Kelvin using [slope, offset] of [0.4706,190.0]

.samples.co.tif: a COG with a single band containing sample position for the OLS pixel along the swath.

Data range: [1,1465]

No-data value: 0

Data type: 16-bit integer

Units: unitless

.li.co.tif: a COG with a single band containing Lunar Illuminance (LI) values

Data range: [0, inf)

No-data value: -1.0

Data type: floating point

Units: lux
```

**Comment by AlessandroScremin** [comment]
@aapopescu @santilland @Schpidi 
please fisrt of all I never heard about these datasets or that I have to handle them so far....so if I have to do something, please keep me in the loop and not assign me to something I'm not aware of

Then what i have to do? ingest (don't understood what) into EDC as a COG file? or what else?

**Comment by santilland** [comment]
Hello @AlessandroScremin maybe i should have assigned it first to @aapopescu for discussion (i changed it now). I am just going through all the High priority tickets to see what can/should be done, or what has to be done before we can actually do something.
In this case if i understood correctly the initial request came from Patrick.
If we want to have this data somehow included in the dashboard we will have to load them to SH to have a WMS interface to the data.

**Comment by AlessandroScremin** [comment]
@santilland  Hi daniel

I saw high priority and assigned to me...so I was quite surprised....hope my reaction did't "hurt" nobody...was only to clearly understood what was needed.... :)  thnaks for the clarification....

**Comment by santilland** [comment]
Sure, no problem, should have probably made it more clear and maybe waited to discuss it directly

**Comment by santilland** [comment]
Hello @AlessandroScremin i think @aapopescu already communicated this to you, it seems integrating this data has highest priority right now.
Could you give me the layer id once the data has been integrated to SH?

**Comment by santilland** [comment]
Hello @AlessandroScremin and @aapopescu just to recapitulate the current status, my understanding is that it seems the datasets are very large and it not clear what should be imported and how long it would take?
Do we have some ETA on when we think data could be available? If it will take longer to be able to import this data we should consider moving this issue to low priority again?

**Comment by AlessandroScremin** [comment]
@santilland @aapopescu 
It definitely should be put in low priority since the data are more complex than expected, and could not be imported in EDC as they are. Also Synergise had problem ingesting them. We agree with Patrick to stop the activity on this dataset for the moment.

Kr

Alessandro

**Comment by lubojr** [comment]
Closing for now. Put on Hold
